modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/auth/SecondaryStoreUserLoginTestCase.java

```diff
@@ -245,4 +273,59 @@ private void stopTomcat() throws LifecycleException {
         tomcat.destroy();
         LOG.info("Tomcat server stopped.");
     }
+
+    private void addSecondaryJDBCUserStore() throws Exception {
+
+        //creating database
+        H2DataBaseManager dbmanager = new H2DataBaseManager("jdbc:h2:" + ServerConfigurationManager.getCarbonHome()
+                + "/repository/database/" + USER_STORE_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD);
+        dbmanager.executeUpdate(new File(ServerConfigurationManager.getCarbonHome() + "/dbscripts/h2.sql"));
+        dbmanager.disconnect();
+
+        // Register a secondary user store
```

Comment:
Shall we add the missing full stop here?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096